locales/en.yml

```diff
@@ -153,3 +153,6 @@ en:
         status:
           running: running
           stopped: stopped
+        unsupported_service: |-
+          Sorry, service '%{service}' is not supported.
```

Comment:
@bexelbie Is this error message fine. It will appear as (in red):
```
➜  vagrant-service-manager git:(fix-250) ✗ bevsm status abcd
Sorry, service 'abcd' is not supported.
Supported services are docker, openshift and kubernetes.
```